main.cpp

```diff
@@ -46,6 +47,29 @@ int main(int argc, char *argv[])
     application.setOrganizationName("Yubico");
     application.setOrganizationDomain("com.yubico");
 
+    QDesktopWidget * desktop = QApplication::desktop();
+
+    // Get x and y coordinates of all monitors
+    QVariantList monitorList;
+    for (qint32 i = 0, screens = desktop->screenCount(); i < screens; i++)  {
+        QRect monitor = desktop->screenGeometry(i);
+
+        QVariantMap x;
+        QVariantMap y;
+
+        int monitor_x_min = monitor.x();
+        int monitor_x_max = monitor_x_min + monitor.width();
+        x.insert("first", monitor_x_min);
```

Comment:
what if there's 3 (or more) monitors?

Reply:
> what if there's 3 (or more) monitors?

That does not matter. In this code we are storing the coordinates of every monitor. Every monitor has xmin, xmax, ymin, and ymax, since they are rectangles.

Reply:
With that, we can also put both `xMin`/`xMax` and `yMin`/`yMax` in the same `QVariantMap` instead of having to add two per monitor.